packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -10,6 +10,10 @@ export class SankeyChartBase extends React.Component<
   {
```

Comment:
Hovering over streams should highlight that stream. 
What is the opacity of a stream in case the node is highlighted?

Reply:
The Stream opacity should be 80%, i am doing that as part of stream hover. Currently i am working on that.